data/apps/liberapay.toml

```diff
@@ -0,0 +1,4 @@
+name="Liberapay"
+appsURL="https://en.liberapay.com/mehrad.mahmoudian/edit/elsewhere"
```

Comment:
well that's your account ^^

Reply:
I should have paid attention :laughing: Thanks for spotting it. I never thought in million years that they put the account name in the URL of the settings page.

I went back and tried to use some wildcards and etc, but I it seems there is no fix for this. Just reject the PR